tasking/filters.py

```diff
@@ -40,21 +39,21 @@ class TaskFilterSet(filters.FilterSet):
     """
     Filterset for Task
     """
-    date = filters.DateFilter(
-        name='date',
-        lookup_expr=DATETIME_LOOKUPS,
-        method='filter_timing'
-    )
-    start_time = filters.TimeFilter(
-        name='start_time',
-        lookup_expr=TIME_LOOKUPS,
-        method='filter_timing'
-    )
-    end_time = filters.TimeFilter(
-        name='end_time',
-        lookup_expr=TIME_LOOKUPS,
-        method='filter_timing'
-    )
+    date = filters.LookupChoiceFilter(
+        field_name='date',
+        field_class=forms.DateField,
+        lookup_choices=DATETIME_LOOKUPS,
+        method='filter_timing')
+    start_time = filters.LookupChoiceFilter(
+        field_name='start_time',
+        field_class=forms.TimeField,
+        lookup_choices=TIME_LOOKUPS,
+        method='filter_timing')
+    end_time = filters.LookupChoiceFilter(
```

Comment:
The Usage of this changes how api requests should be made. With the use of a LookupChoiceFilter request will now look like this: `https://example.com/api/v1/task/?end_time=example_time&end_time_lookup=exact`

Reply:
Currently looking onto ways to change the filterset to work with the usual `https://example.com/api/v1/task/?end_time_gt=exampletime`.